components/shared/workCard/WorkCard.tsx

```diff
@@ -41,7 +57,12 @@ const WorkCard = ({ work }: WorkCardProps) => {
   return (
     <div className="mb-4">
       <Link href={`/work/${work.workId}`}>
-        <div key={work.workId} className="rounded-sm bg-background-overlay p-2 md:p-4">
+        <div key={work.workId} className="relative rounded-sm bg-background-overlay p-2 md:p-4">
+          {!!dataPublizon?.product?.costFree && (
+            <Badge variant={"blue"} className="absolute left-2 md:left-4 md:top-4">
+              BLÅ
+            </Badge>
+          )}
           <div
             className="px-auto relative mx-auto mb-3 mt-6 flex aspect-[166/228] w-[calc(100%-76px-4px)] items-center
```

Comment:
- `px-auto` is not at valid class
- consider width and padding to be a bit more dynamic w-[80%]  
- I suggest removing aspect ratio and create a container where the image can be placed inside and use its own intrinsic image dimensions 

Something like this

```suggestion
            className="relative mx-auto mb-3 mt-6 flex w-[80%] items-center
```

Reply:
I removed the padding class :)))

Without the aspect, the cards will collapse depending on the cover, which doesn't look desireable:
![image](https://github.com/user-attachments/assets/7bf88068-aa67-4549-9724-e508b2c4ffbc)
![image](https://github.com/user-attachments/assets/4977ea19-b49a-4e44-9db7-68b306285bdb)
I'm keeping the aspect ratio :3

Reply:
Okay didn't expect that to happen. Keep the aspect ratio and simplify it?  

Maybe try whatever is closest of the already defined aspect ratios? 
We have those in the config. If none fit then create a new one. 

 aspectRatio: {
    "1/1": "var(--aspect-1-1)",
    "1/2": "var(--aspect-1-2)",
    "2/1": "var(--aspect-2-1)",
    "2/3": "var(--aspect-2-3)",
    "3/2": "var(--aspect-3-2)",
    "3/4": "var(--aspect-3-4)",
    "4/3": "var(--aspect-4-3)",
    "4/5": "var(--aspect-4-5)",
    "5/2": "var(--aspect-5-2)",
    "5/4": "var(--aspect-5-4)",
    "16/9": "var(--aspect-16-9)",
  },